OMCompiler/Compiler/Util/Error.mo

```diff
@@ -1032,6 +1032,8 @@ public constant Message UNBOUND_PARAMETER_EVALUATE_TRUE = MESSAGE(594, TRANSLATI
   Util.gettext("Parameter %s has annotation(Evaluate=true) and no binding."));
 public constant Message FMI_URI_RESOLVE = MESSAGE(595, TRANSLATION(), WARNING(),
   Util.gettext("Could not resolve URI (%s) at compile-time; copying all loaded packages into the FMU"));
+public constant Message PATTERN_MIXED_POS_NAMED = MESSAGE(596, TRANSLATION(), WARNING(),
+  Util.gettext("Call to %s contains mixed positional and mixed arguments."));
```

Comment:
"mixed positional and named arguments"???

Reply:
Yeah, that should be it -_-